model/comment.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose")
+
+// schema for the comment
+const commentSchema = mongoose.Schema({
+    
+        content: {
+            type: String,
+            required: true
+        },
+        post: {
+            type: mongoose.Types.ObjectId
+        },
```

Comment:
`ref`, and `required` are missing

Reply:
I ref post and for the required I dont think I need it here cause its commenting should be optional to the user. I also added to the post schema now.